src/app/frontend/logs/logstoolbar/logstoolbar_controller.js

```diff
@@ -130,7 +130,14 @@ export default class LogsToolbarController {
    * @return {!backendApi.ReplicationControllerPodWithContainers|undefined}
    * @private
    */
-  findPodByName_(array, name) { return array.find((element) => element.name === name); }
+  findPodByName_(array, name) {
```

Comment:
Modifying built in classes is a controversial topic, and, personally, I'd avoid it at any cost. You can find many articles on it on the net, e.g., https://www.nczonline.net/blog/2010/03/02/maintainable-javascript-dont-modify-objects-you-down-own/

If this incompatibility hurts us, we can fall back to closure library wrappers (e.g., goog.array.find) Or use closure compiler to polyfil missing function (e.g., Array.prototype.find). But, if possible, I'd like us to run with Babel on all major browsers (ie, chrome, ff).
What do you think?